### PR TITLE
Reinstate AWS production deployments

### PR DESCRIPTION
**Pull request by adammcmaster**
When the `production-release` tag is pushed:
- Push `zooniverse/panoptes:production-release` to Docker Hub
- Build AMIs for panoptes-api and panoptes-dumpworker
- Deploy panoptes-api and panoptes-dumpworker

# Review checklist

- [ ] First, the most important one: is this PR small enough that you can actually review it? Feel free to just reject a branch if the changes are hard to review due to the length of the diff.
- [ ] If there are any migrations, will they the previous version of the app work correctly after they've been run (e.g. the don't remove columns still known about by ActiveRecord).
- [ ] If anything changed with regards to the public API, are those changes also documented in the `apiary.apib` file?
- [ ] Are all the changes covered by tests? Think about any possible edge cases that might be left untested.

### CODE REVIEW

**Comment by adammcmaster** [comment]
The updated environment variables file is in place, and I've tested it by configuring an EC2 instance using the Docker image currently deployed to Kubernetes.

File: Jenkinsfile [hunk 1]
```diff
@@ -42,10 +42,149 @@ pipeline {
...
+                  message: "FAILED: Job '${env.JOB_NAME} [${env.BUILD_NUMBER}]' (${env.BUILD_URL})",
+                  channel: "#ops"
+                  )
+            }
+          }
         }
       }
     }
 
+    stage('Deploy production AMIs') {
```
Comment (camallen) [inline Jenkinsfile:133]: I don't think we want this deploy script to automatically run on `production-release` tag (migrations). We will want to run the deploy via the lita cmd as per the previous setup.

Comment (adammcmaster) [reply Jenkinsfile:133]: My aim is to have this deploy using the newer `lita deploy` command, which works by updating the tag to trigger the deploy. I'm not trying to get the old `lita panoptes deploy` command working again.

Comment (camallen) [reply Jenkinsfile:133]: our current issue is running migrations as they will need to run before the code deploys to ensure the db schema matches the code ORM schema. If we can finalize the migration jobs running correctly (on their own tag) then i think we can safely pursue this deploy path, i.e. 
1. merge to master deploy to staging
2. run a migration job by updating `production-migrate` tag
3. update the `production-release` tag to trigger the prod deployment

**Pull request merged**